### PR TITLE
1220 display lloq in time profiles

### PR DESCRIPTION
**Pull request by Felixmil**
```
dataSet <- DataSet$new("ds with lloq")
dataSet$setValues(1:7, c(10 * exp(-1:-7) + rnorm(7, 0, .25)), abs(rnorm(7, 0, 0.1)))
dataSet$LLOQ <- 0.15

dc <- DataCombined$new()
dc$addDataSets(dataSet)
```

## With lloq (default)

```r
plotIndividualTimeProfile(dc)
```

image

## Without lloq

We can use the new setting `displayLLOQ`  in ` DefaultPlotConfiguration` to not plot LLOQ lines

```r
  noLLOQ <- DefaultPlotConfiguration$new()
  noLLOQ$displayLLOQ <- FALSE

plotIndividualTimeProfile(dc, defaultPlotConfiguration = noLLOQ)
```
image

### CODE REVIEW

File: tests/testthat/_snaps/plot-individual-time-profile/geometric-error.svg
[the remark this answers is not in the document]
Comment (Felixmil) [reply tests/testthat/_snaps/plot-individual-time-profile/geometric-error.svg:1]: I  executed this the `develop` branch, in debug mode while executing the test for this plot:
```r
tlf::plotTimeProfile(
    data = simData,
    dataMapping = simulatedDataMapping,
    observedData = obsData,
    observedDataMapping = tlf::ObservedDataMapping$new(x, y, ymin, ymax,
                                                       shape = shape,
                                                       color = color,
                                                       lloq = "lloq"
    ),
    plotConfiguration = timeProfilePlotConfiguration
)
```

it generated this:
image

Here I just changed calls to {tlf} so the bug comes from there. I'll open an issue.

[the remark this answers is not in the document]
Comment (Felixmil) [reply tests/testthat/_snaps/plot-individual-time-profile/geometric-error.svg:1]: LLOQ column was always mapped even with there was no value in it. Now, the line appears around the symbol only if lloq column of obsData is not empty.

[the remark this answers is not in the document]
Comment (Felixmil) [reply tests/testthat/_snaps/plot-individual-time-profile/geometric-error.svg:1]: should the `plotIndividualTimeProfile()` have an argument to choose whether the lloq lines should be displayed or not ?

[the remark this answers is not in the document]
Comment (PavelBal) [reply tests/testthat/_snaps/plot-individual-time-profile/geometric-error.svg:1]: > should the plotIndividualTimeProfile() have an argument to choose whether the lloq lines should be displayed or not ?

Maybe it should be a property of the `DefaultPlotConfiguration`, and respected by all `plot...()` function that support LLOQ?

> LLOQ column was always mapped even with there were no values in it. Now, the line appears around the symbol only if lloq column of obsData is not empty.

Ok, got it. Can this be changed? Having an LLOQ line should not result in a line in the legends symbol.

[the remark this answers is not in the document]
Comment (Felixmil) [reply tests/testthat/_snaps/plot-individual-time-profile/geometric-error.svg:1]: > Ok, got it. Can this be changed? Having an LLOQ line should not result in a line in the legends symbol.
Yes, in TLF

[the remark this answers is not in the document]
Comment (PavelBal) [reply tests/testthat/_snaps/plot-individual-time-profile/geometric-error.svg:1]: > Yes, in TLF

Great, please create an issue.

[the remark this answers is not in the document]
Comment (Felixmil) [reply tests/testthat/_snaps/plot-individual-time-profile/geometric-error.svg:1]: ✅  https://github.com/Open-Systems-Pharmacology/TLF-Library/issues/453

**Review by PavelBal — APPROVED** [review]

**Review by PavelBal — APPROVED** [review]

**Comment by Felixmil** [comment]
>>Maybe it should be a property of the DefaultPlotConfiguration, and respected by all plot...() function that support LLOQ?
>
>Roger that, i'll add this inside this PR.

This has been added. See [top comment](#issue-1652342474) for previesw.

**Comment by PavelBal** [comment]
>This has been added. See [top comment](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/1228#issue-1652342474) for preview.

Just wondering... Are there two different data sets you are using for the preview with vs without LLOQ?

**Comment by Felixmil** [comment]
> Just wondering... Are there two different data sets you are using for the preview with vs without LLOQ?

This was because of some `rnorm` in the dataset generation and I forgot to update the first image. Should look fine now.

**Review by PavelBal — APPROVED** [review]

**Pull request merged**